test/replutil.jl

```diff
@@ -227,8 +227,9 @@ let err_str,
     i = reinterpret(EightBitType, 0x54),
     j = reinterpret(EightBitTypeT{Int32}, 0x54)
 
-    err_str = @except_str Symbol() MethodError
-    @test contains(err_str, "MethodError: no method matching Symbol()")
+    #PR 16130, replaced Symbol() with Float64()
```

Comment:
This comment isn't useful here. We don't want to add a comment above each line to recall it's history (and git is there for this).